src/os/resourcesdir.zig

```diff
@@ -2,13 +2,41 @@ const std = @import("std");
 const builtin = @import("builtin");
 const Allocator = std.mem.Allocator;
 
+pub const ResourcesDir = struct {
+    app_path: ?[]const u8 = null,
+    host_path: ?[]const u8 = null,
```

Comment:
We should put a comment above these that note callers should prefer to use the accessor functions rather than directly accessing these fields.